### DISCUSSION TITLE
how to update slidev?

### DISCUSSION THREAD

**Discussion by zillionare** [slidevjs/slidev#1862] [Q&A]
I'm now using slidev@v0.49.16, I want to update it to v0.49.29. I have made update successfully before but I forget the method.

Doesn't anywhere in slidev documentation mentions how to update slidev?

**Comment by kermanx** [comment]
`pnpm i slidev@0.49.29`?

**Reply by zillionare** [reply]
can't find in registry.

I have updated slidev by the command:

```
npm install @slidev/cli@latest
```

Not sure if it's correct, but it already made my project not working anymore...
